scripts/criu-ns

```diff
@@ -139,6 +155,23 @@ def wrap_restore():
         _mount_new_proc()
         run_criu(restore_args)
 
+    if restore_pidfile:
+        restored_pid = None
+        retry = 5
+
+        while not restored_pid and retry:
+            with open('/proc/%d/task/%d/children' % (criu_pid, criu_pid)) as f:
+                line = f.readline().strip()
+                if len(line):
+                    restored_pid = line
```

Comment:
question: I have executed this version of `criu-ns` script multiple times with my tests. And I always ended up with the same (content of `pidfile`) `restored_pid`: `15`. Is this right? :)

Reply:
I believe this is because we get the value of `restored_pid` after unsharing the PID namespace in `criu-ns`.

Reply:
Unsharing pid namespace does not change current pid namespace, it only changes pid namespace for children, current implementation in criu-ns in this pr gives pid in pidns of the caller, if you have pid 15 there each time maybe you caller is itself in some pidns...

Reply:
understood, thanks @Snorch. Yes, I can dump pid 15 :)